CleverCloud/clever-components
# lit: upgrade to 3.2.1

Fixes #1203 

## What does this PR do?

- Upgrades `lit` from `2.3.1` to `3.2.1` (no impact on our code, see [Lit 3 upgrade guide](https://lit.dev/docs/releases/upgrade/) for more info),
- Upgrades dependencies that also depend on `lit` to latest:
  - `@open-wc/testing` from `3.1.6` to `4.0.0` (no impact on our code, see [@open-wc/testing releases](
https://github.com/open-wc/open-wc/releases?q=open-wc%2Ftesting&expanded=true) for more info),
  - `@lit-labs/motion` from `1.0.3` to `1.0.7` (no impact on our code, see [\@lit-labs/motion releases](
https://github.com/lit/lit/releases?q=lit-labs%2Fmotion&expanded=true) for more info),
  - `@lit-labs/virtualizer` from `2.0.12` to `2.0.14` (no impact on our code, see [\@lit-labs/virtualizer releases](
https://github.com/lit/lit/releases?q=virtualizer&expanded=true) for more info).

## How to review?

- Check the toaster stories in preview since they rely on `@lit-labs/motion`,
- Check the logs & kv explorer stories in preview since they rely on the virtualizer,
- Run `npm ls lit` or `npm ls @lit/reactive-element`: you should see that only Shoelace brings a different lit version (already fixed with #1218)
- CI mostly does the job for the rest,
- 2 reviewers should be enough for this one.
